### PR TITLE
Docs/initial setup

### PR DESCRIPTION
**Pull request by rishabh3112**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,27 @@
...
+## Initial Setup
+1. Initialize `npm`
+```shell
+$ npm init 
+```
+2. Install `webpack` and `webpack-cli`
+```shell
+$ npm install --save-dev webpack webpack-cli
+```
+3. Install `@webpack-cli/init`
```
[the remark this answers is not in the document]
Comment (rishabh3112) [reply README.md]: `init` @sendilkumarn.

[the remark this answers is not in the document]
Comment (sendilkumarn) [reply README.md]: Then why it is `scaffold`  below 😉

[the remark this answers is not in the document]
Comment (sendilkumarn) [reply README.md]: Let me amend it 👍

[the remark this answers is not in the document]
Comment (rishabh3112) [reply README.md]: Ohh typo error 🙈

**Pull request merged**